server/writer/gzip.go

```diff
@@ -93,3 +96,5 @@ func ModifyAcceptEncoding(header http.Header) {
 		header.Del(AcceptEncodingHeader)
 	}
 }
+
+func (g *Gzip) AddModifier(_ *eval.Context, _ []hcl.Body) {}
```

Comment:
Should be removed here. Just define this Method as own interface instead of using the `writer` if required. As I can see, there are explicit type assertions instead of using the `writer` interface.